admin/registration_providers/views.py

```diff
@@ -32,6 +32,7 @@ def form_valid(self, form):
         self.object = form.save(commit=False)
         self.object._creator = self.request.user
         self.object.save()
+        self.object.update_group_permissions()  # Creating moderator groups
```

Comment:
While is technically correct, it shouldn't really be necessary due to [this line](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/provider.py#L333-L337) which __just__ got merged. Could you pull from develop and test whether the problem fixed itself?

Reply:
Confirmed. Closing PR